src/uct/api/v2/uct_v2.h

```diff
@@ -577,6 +596,47 @@ typedef struct uct_iface_is_reachable_params {
 } uct_iface_is_reachable_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_ep_is_connected.
+ *        Only valid addresses which are present in remote_address should be
```

Comment:
What is `remote_address`? It is not "known" in this context.

Reply:
not sure what to do here